src/layouts/ha-app-layout.js

```diff
@@ -88,12 +100,22 @@ class HaAppLayout extends customElements.get('app-header-layout') {
     <div id="wrapper" class="initializing">
       <slot id="headerSlot" name="header"></slot>
 
-      <div id="contentContainer">
+      <div id="contentContainer" class$="[[_getTabsHidden(tabsHidden)]]">
```

Comment:
Don't change ha-app-layout. ha-app-layout doesn't have to know anything about tabs or even the content of the header.